app/renderer/settings.js

```diff
@@ -1113,6 +1131,9 @@ const set = (setting, value) => {
         if (downloadSettings.includes(setting)) {
             updateDownloadSettings()
         }
+        if (bookmarkSettings.includes(setting)) {
```

Comment:
This can just directly check for `boomarksfile`, as the other setting can go, and because `suggestorder` updates don't mean that an update is needed in bookmarks.